### PR TITLE
Make sure links have text

### LINKED ISSUE #95: Links must have discernible text
### Steps to recreate
Go to https://static-tables-staging.princeton.edu/
Open the dev console.
Scan the page using axe-core tools browser extension

### URL
https://static-tables-staging.princeton.edu/

### Issues
Ensures links have discernible text

### Severity
High

### WCAG Success Criterion
[SC 2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

[SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

### User impact
Users do not understand the purpose of each link so they can decide whether they want to follow the link.

### Recommended fix
To solve this problem, you need to fix at least (1) of the following:

    Element does not have text that is visible to screen readers
    aria-label attribute does not exist or is empty
    aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
    Element has no title attribute

And fix the following:

Element is in tab order and does not have accessible text

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
[Accessibility audit_05152024_Static-Tables](https://docs.google.com/spreadsheets/d/1_-Khj5TezyEu2gVb2jJczWCUt04bt5_P8-zL0f269kQ/edit#gid=0)

### PR DESCRIPTION
**Pull request by maxkadel**
Closes #95

### CODE REVIEW

File: app/components/Footer.js [hunk 1]
```diff
@@ -107,15 +107,19 @@ export default {
               <li>
                 <a class="footer-icon" href="https://www.facebook.com/PULibrary">
                   <img :src="facebook">
+                  <span class="visually-hidden">Facebook</span>
```
Comment (sandbergja) [inline app/components/Footer.js:110]: I think adding "Facebook" as an alt property for the image would also resolve the issue -- would you be able to give that a try?

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @maxkadel  -- one suggestion of an alternate approach

File: app/components/Footer.js (hunk 1 above)
Comment (maxkadel) [reply app/components/Footer.js]: Done!

**Review by sandbergja — APPROVED** [review]
Thanks, @maxkadel ! 🦄

**Pull request merged**